riak_test/tests/gc_tests.erl

```diff
@@ -206,3 +223,47 @@ verify_riak_object_remaining_for_bad_key(RiakNodes, GCKey, {{Bucket, Key}, UUID}
     lager:info("As expected, BAD manifest in GC bucket remains,"
                " stand off orphan manfiests/blocks: ~p", [Manifest]),
     ok.
+
+verify_partial_gc_run(CSNode, RiakNodes,
+                      {MegaSec0, Sec0, _},
+                      {MegaSec1, Sec1, _}) ->
+    Start0 = MegaSec0 * 1000000 + Sec0,
+    End0 = MegaSec1 * 1000000 + Sec1,
+    Interval = erlang:max(1, (End0 - Start0) div 3),
+    Starts = [ {Start0 + N * Interval, Start0 + (N+1) * Interval}
+               || N <- lists:seq(0, 3) ],
```

Comment:
8 is one of the very happy exceptional cases. I guess this test failed non-deterministic way on slow machine.

```
> [begin Int = End div 3, To = 4 * Int, {End, Int, To, End =:= To, End < To, To < End} end || End <- lists:seq(1,20)].
[{1,0,0,false,false,true},
 {2,0,0,false,false,true},
 {3,1,4,false,true,false},
 {4,1,4,true,false,false},
 {5,1,4,false,false,true},
 {6,2,8,false,true,false},
 {7,2,8,false,true,false},
 {8,2,8,true,false,false},
 {9,3,12,false,true,false},
 {10,3,12,false,true,false},
 {11,3,12,false,true,false},
 {12,4,16,false,true,false},
 {13,4,16,false,true,false},
 {14,4,16,false,true,false},
 {15,5,20,false,true,false},
 {16,5,20,false,true,false},
 {17,5,20,false,true,false},
 {18,6,24,false,true,false},
 {19,6,24,false,true,false},
 {20,6,24,false,true,false}]
```